Gordon360/Services/HousingService.cs

```diff
@@ -31,39 +32,89 @@ public IEnumerable<HousingViewModel> GetAll()
         {
             return RawSqlQuery<HousingViewModel>.query("GET_STU_HOUSING_INFO");
         }
+        */
 
         /// <summary>
         /// Saves student housing info
+        /// - first, it creates a new row in the applications table and puts the id of the primary applicant and a timestamp
+        /// - second, it looks for the application id of the application with the information we just input (because 
+        /// the database creates the application ID so we have to ask it which number it generated for it)
+        /// - third, it inserts each applicant into the applicnts table along with the apartment ID so we know
+        /// which application on which they are an applicant
+        ///  
         /// </summary>
-        /// <returns>The housing item</returns>
-        public IEnumerable<HousingAppToSubmitViewModel>SaveApplication(string username, string sess_cde, string [] appIds)
+        /// <returns>Whether or not all the queries succeeded</returns>
+        public int SaveApplication(int apartAppId, string modifierId, string sess_cde, string [] applicantIds)
         {
-           //return RawSqlQuery<HousingViewModel>.query("INSERT_AA_APPLICATION", id);
-            IEnumerable<HousingAppToSubmitViewModel> result = null;
-            IEnumerable<HousingViewModel> result2 = null;
-            IEnumerable<HousingAppToSubmitViewModel> result3 = null;
-
+            IEnumerable<ApartmentAppSaveViewModel> result = null;
+            IEnumerable<ApartmentAppIDViewModel> idResult = null;
+            IEnumerable<ApartmentApplicantViewModel> result2 = null;
```

Comment:
I think we need better variable names that are less confusing and self-explanatory. Something better than just saying "result" and "result2".

Reply:
I changed the names to 'newApplicationResult' and 'applicantResult'

Reply:
Did you change that in this branch?

Reply:
Yes, commit [fec6462](https://github.com/gordon-cs/gordon-360-api/pull/421/commits/fec6462f81c1c047f40a0d86824cae43f8849667)